traffic_ops/traffic_ops_golang/ats/parentdotconfig.go

```diff
@@ -495,30 +495,6 @@ func GetATSMajorVersion(tx *sql.Tx, serverProfileID ProfileID) (int, error) {
 	return atsMajorVer, nil
 }
 
-// GetProfileParamValue gets the value of a parameter assigned to a profile, by name and config file.
-// Returns the parameter, whether it existed, and any error.
-func GetProfileParamValue(tx *sql.Tx, profileID ProfileID, configFile string, name string) (string, bool, error) {
```

Comment:
I think you removed this but forgot to bring it back in from your other PR?
```
ats/parentdotconfig.go:483:24: undefined: GetProfileParamValue
```

Reply:
Fixed